keyboards/gh60/keymaps/zorched

```diff
@@ -0,0 +1 @@
+../../../../qmk_gh60_zorched
```

Comment:
This file should be removed from the Pull Request.

Reply:
Good catch. Sorry was sloppy.